gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/replication/ReplicationConfiguration.java

```diff
@@ -193,6 +201,11 @@ public Builder withEnforceFileSizeMatchFromConfigStore(Config config) {
       return this;
     }
 
+    public Builder withSchemaCheckEnabled(Config config) {
+      this.schemaCheckEnabled = !config.hasPath(COPY_SCHEMA_CHECK_ENABLED) || config.getBoolean(COPY_SCHEMA_CHECK_ENABLED);
```

Comment:
Consider to use ConfigUtils.getConfig(config, COPY_SCHEMA_CHECK_ENABLED, DEFAULT_COPY_SCHEMA_CHECK_ENABLED)